package.json

```diff
@@ -5,7 +5,9 @@
   "main": "./dist/index.js",
   "scripts": {
     "webpack": "node ./node_modules/webpack/bin/webpack --config webpack.config.js --display-error-details --progress --profile --color",
-    "build": "npm run webpack && extractInfo && copyfiles -f ./gameart.png ./dist/",
+    "build": "npm run webpack && extractInfo && copyfiles -f ./gameart.png ./gameicon.png ./LICENSE ./dist/",
+    "pack": "npm run build && powershell ./script/package.ps1",
+    "pack-dev": "npm run build && powershell ./script/package.ps1 -Dev",
```

Comment:
Can `powershell` be `powershell.exe` instead? unless there is some npm magic going on, I don't think this will work on WSL.